### DISCUSSION TITLE
[bug] YetiPortal2 install doc

### DISCUSSION THREAD

**Discussion by guanchgonzalez** [YetiForceCompany/YetiForceCRM#14986] [How to] [answered]
Hi!

Could you share the installation docu of YetiPortal2?

Many thanks in advance.

**Comment by rskrzypczak** [comment, marked as answer]
In simple words ..
1. Download [YetiForcePortal2-6.2-complete.zip](https://github.com/YetiForceCompany/YetiForcePortal2/releases) 
2. Put files on web server
3. Point web server document root to public_html directory
4. Launch the page

https://github.com/YetiForceCompany/YetiForcePortal2#-installation
